Bottom: 291b34c8e49d209a61e6c74a76affcc809c4b9a1
Top:    8c4223ffc3f44d360ef7c5cc7797c97e6d84b108
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-27 00:37:26 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 7b0769d..7ac8233 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -329,13 +329,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
